src/commands/build.rb

```diff
@@ -32,24 +32,27 @@
 require 'nodes'
 require 'output'
 require 'build_files_retriever'
+require 'exceptions'
 
 module Metalware
   module Commands
     class Build < CommandHelpers::BaseCommand
       private
 
-      attr_reader :group_name, :nodes
+      attr_reader :group_name, :nodes, :edit_start, :edit_continue
 
       delegate :template_path, to: :file_path
 
       def setup
+        setup_edit_mode
         node_identifier = args.first
         @group_name = node_identifier if options.group
         @nodes = Nodes.create(config, node_identifier, options.group)
       end
 
       def run
-        render_build_templates
+        render_build_templates unless edit_continue
+        return if run_edit_start
```

Comment:
Rather than having a conditional inside this conditional, it could be clearer to do something like this:

```ruby
if edit_start
  puts EDIT_START_MESSAGE
  return
end
```

You could make this a one-liner which does the same thing if you prefer, but either seems a bit clearer than checking `edit_start` twice in two places to me.